packages/server/graphql/mutations/helpers/generateWholeMeetingSummary.ts

```diff
@@ -33,7 +37,12 @@ const generateWholeMeetingSummary = async (
   if (contentToSummarize.length <= 1) return
   const [r, summary] = await Promise.all([getRethink(), manager.getSummary(contentToSummarize)])
   if (!summary) return
-  await r.table('NewMeeting').get(meetingId).update({summary}).run()
+  const [meeting] = await Promise.all([
+    dataLoader.get('newMeetings').load(meetingId),
+    r.table('NewMeeting').get(meetingId).update({summary}).run()
+  ])
+  // mutate the cache
+  meeting.summary = summary
```

Comment:
Alternatively, I could return the summary here and pass it to `updateQualAIMeetingsCount`, but I prefer to pass fewer params around and fetch the `meeting` with the `summary` in `updateQualAIMeetingsCount`

Reply:
-1 I'm not really happy with the load an object just to update the cache approach. If it wasn't loaded or isn't needed later anymore, then we've just wasted some bandwidth and time. I think I've mentioned this before, but I would prefer if we'd had something like
```
dataLoader.get('newMeetings').updateCache(meetingId, {summary})
```
which we could put in the `Promise.all` call and which would update the cache only if it's in there.

Marking it as -1 to avoid the current approach to become a pattern.

Reply:
I think that's a better approach but can be handled in a separate issue to help get this shipped. I've created an issue here: https://github.com/ParabolInc/parabol/issues/7687